aries-backchannels/afgo/afgo_backchannel.py

```diff
@@ -174,6 +174,7 @@ def __init__(
         self.proofOperationTranslationDict = {
             "prepare-json-ld": "",
             "send-proposal": "send-propose-presentation",
+            "accept-proposal": "accept-propose-presentation",
             "send-request": "send-request-presentation",
```

Comment:
What's the difference between accept-proposal and send-request?

Reply:
Present-proof v3, like v2, can start with a request message or a proposal - `send-request` starts a new thread, starting the protocol with the request, while `accept-proposal` continues the thread started by a proposal.

Reply:
Just addressed this, can you review?